package.json

```diff
@@ -32,7 +32,7 @@
     "install:affinidi-java-springboot": "cd samples/affinidi-java-springboot && sh mvnw install",
     "start": "concurrently -n w: \"npm:start:*(!template)\"",
     "start:template": "npm run --prefix generator/template dev -- -p 3000",
-    "start:affinidi-nextjs-nextauthjs": "npm run --prefix samples/affinidi-nextjs-nextauthjs dev -- -p 3001",
+    "start:affinidi-nextjs-nextauthjs": "npm run --prefix samples/affinidi-nextjs-nextauthjs dev -- -p 3000",
```

Comment:
revert this one?

Reply:
Should we then maybe change `.env.example` to have `NEXTAUTH_URL="http://localhost:3000"`?

Reply:
Thats already the case. The thing is that when you run the generator it updates the samples' `.env` files to change their port so that they dont collide when running multiple 

https://github.com/affinidi/reference-app-affinidi-vault/blob/iota-feature/generator/nextjs/generate.mjs#L91

Reply:
Aha! Gotcha, it has been reverted.